### PR TITLE
added home page

### PR DESCRIPTION
**Pull request by Hamza-777**
Added home page to my ecomm website please review and provide feedback.
live: https://624012f9a8b5c6538766df29--aquamarine-liger-797ab3.netlify.app/

### CODE REVIEW

File: src/Components/Pages/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                    <Link to="/product-listing" className="btn btn-to flex-center">&#10095;</Link>
+                    <div className="fader"></div>
+                </div>
+            </div>
+            <div className="trendings flex-center flex-col align-start">
+                <h1>🔥 Trendings 🔥</h1>
+                <div className="trending-games flex align-center">
+                    {
+                        products && products.map(product => (
+                            <Card key={product._id} id={product._id} imgSrc={product.imgSrc} title={product.title} price={product.price} stars={product.stars} />
```
Comment (nayyyhaa) [inline src/Components/Pages/Home.jsx:24]: you can destructure product here for more cleaner code.

File: src/Components/Providers/ProductProvider.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import axios from 'axios';
+
+const productContext = createContext(null);
+
+const ProductProvider = ({ children }) => {
+    const [ products, setProducts ] = useState([]);
+
+    useEffect(() => {
+        (
+            async () => {
```
Comment (nayyyhaa) [inline src/Components/Providers/ProductProvider.jsx:11]: appreciation: good that you used IIFE function here!

**Review by nayyyhaa — APPROVED** [review]
Well-written code with proper folder structure. not much to comment on since only the basic homepage is integrated. You can merge it. 👍

File: src/Components/Pages/Home.jsx (hunk 1 above)
Comment (Hamza-777) [reply src/Components/Pages/Home.jsx:24]: Yes that would be nice, will do that, thank you.

File: src/Components/Providers/ProductProvider.jsx (hunk 2 above)
Comment (Hamza-777) [reply src/Components/Providers/ProductProvider.jsx:11]: Thanks

**Pull request merged**